oss-internship-2020/libtiff/test/data.h

```diff
@@ -0,0 +1,31 @@
+#include <array>
+
+constexpr uint32_t kRawTileNumber = 9;
+constexpr uint32_t kClusterSize = 6;
+constexpr uint32_t kChannelsInPixel = 3;
+constexpr uint32_t kTestCount = 3;
+constexpr uint32_t kImageSize = 128 * 128;
+constexpr uint32_t kClusterImageSize = 64 * 64;
```

Comment:
These should be `inline constexpr`, also below.

For details, see https://abseil.io/tips/140 under "Constants in header files".